src/Shared/DiagnosticIds/DiagnosticIds.cs

```diff
@@ -86,6 +86,8 @@ internal static class LoggerMessage
         internal const string LOGGEN034 = nameof(LOGGEN034);
         internal const string LOGGEN035 = nameof(LOGGEN035);
         internal const string LOGGEN036 = nameof(LOGGEN036);
+        internal const string LOGGEN037 = nameof(LOGGEN037);
+        internal const string LOGGEN038 = nameof(LOGGEN038);
```

Comment:
Need to update https://github.com/dotnet/extensions/blob/main/docs/list-of-diagnostics.md and add an aka.ms entry.

Reply:
Thanks, missed that!